LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/NetworkServerConfiguration.cs

```diff
@@ -145,6 +145,8 @@ public class NetworkServerConfiguration
         /// </summary>
         public int ProcessingDelayInMilliseconds { get; set; } = Constants.DefaultProcessingDelayInMilliseconds;
 
+        public bool IsLocalDevelopment { get; set; }
```

Comment:
I would say that if we don't use it, we should remove it and eventually add later on.
But it's also fine if you want to keep here and use it in the future

Reply:
Done